sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -337,69 +337,6 @@ class RocksDBSuite extends SparkFunSuite {
     }
   }
 
-  ignore("ensure that concurrent update and cleanup consistent versions") {
-    quietly {
-      val numThreads = 20
-      val numUpdatesInEachThread = 20
-      val remoteDir = Utils.createTempDir().toString
-      @volatile var exception: Exception = null
-      val updatingThreads = Array.fill(numThreads) {
-        new Thread() {
-          override def run(): Unit = {
-            try {
-              for (version <- 0 to numUpdatesInEachThread) {
-                withDB(
-                  remoteDir,
-                  version = version) { db =>
-                  val prevValue = Option(toStr(db.get("a"))).getOrElse("0").toInt
-                  db.put("a", (prevValue + 1).toString)
-                  db.commit()
-                }
-              }
-            } catch {
-              case e: Exception =>
-                val newException = new Exception(s"ThreadId ${this.getId} failed", e)
-                if (exception != null) {
```

Comment:
First all, `exception` is null at beginning. So this condition will be never satisfied.